### ISSUE TITLE
number of trajectories are zero

### ISSUE THREAD

**Issue by ahellander** [Molns/molns#53]
In running distributed ensembles and subsequently a mean caclulation, I keep getting this error. 

---

MolnsUtilException                        Traceback (most recent call last)
<ipython-input-19-f8bdbbbc96e7> in <module>()
     11     # Phase 2 - read all the realizations and compute the mean value
     12     tic = time.time()
---> 13     res = ensemble.mean(mapper=g)
     14     wall_time_2 = time.time()-tic
     15     wall_time_strong_phase2.append(wall_time_2)

/usr/local/lib/python2.7/dist-packages/molnsutil-0.8-py2.7.egg/molnsutil/molnsutil.pyc in mean(self, mapper, number_of_trajectories, chunk_size, verbose, store_realizations, storage_mode, cache_results)
   1027         """ Compute the mean of the function g(X) based on number_of_trajectories realizations
   1028             in the ensemble. It has to make sense to say g(result1)+g(result2). """
-> 1029         return self.run(mapper=mapper, aggregator=builtin_aggregator_add, reducer=builtin_reducer_mean, number_of_trajectories=number_of_trajectories, chunk_size=chunk_size, verbose=verbose, store_realizations=store_realizations, storage_mode=storage_mode, cache_results=cache_results)
   1030 
   1031 

/usr/local/lib/python2.7/dist-packages/molnsutil-0.8-py2.7.egg/molnsutil/molnsutil.pyc in run(self, mapper, aggregator, reducer, number_of_trajectories, chunk_size, verbose, progress_bar, store_realizations, storage_mode, cache_results)
    749             raise MolnsUtilException("Storage mode already set to {0}, can not mix storage modes".format(self.storage_mode))
    750         if number_of_trajectories is None or number_of_trajectories == 0:
--> 751             raise MolnsUtilException("'number_of_trajectories' is zero.")
    752         elif not store_realizations and self.number_of_trajectories > 0 and self.number_of_trajectories != number_of_trajectories:
    753             raise MolnsUtilException("'number_of_trajectories' changed since first call.  Value can only be changed if store_realizations is True")

MolnsUtilException: 'number_of_trajectories' is zero.

**Comment by ahellander** [comment]
Is this addressed in the new version of briandrawert/molnsutil_state?

**Comment by briandrawert** [comment]
Yes, this should be addressed by https://github.com/briandrawert/molnsutil/commit/634a1088073a9371fa4fbeda410b9f6100d90ef9, see line 805
